doc/userguide/install.rst

```diff
@@ -66,52 +66,102 @@ Dependencies
 
 For Suricata's compilation you'll need the following libraries and their development headers installed::
 
-  libjansson, libpcap, libpcre2, libmagic, zlib, libyaml
+  libjansson, libpcap, libpcre2, libyaml, zlib
 
 The following tools are required::
 
-  make gcc (or clang) pkg-config
-
-For full features, also add::
-
-  libgeoip, liblua5.1, libhiredis, libevent
+  make gcc (or clang) pkg-config rustc cargo
 
 Rust support::
 
   rustc, cargo
 
-  Not every distro provides Rust packages yet. Rust can also be installed
-  directly from the Rust project itself::
+  Some distros provide outdated/don't provide Rust packages.
```

Comment:
sentence can be a bit clearer

Reply:
changed to "Some distros don't provide or provide outdated Rust packages."